QRCoder/QRCodeGenerator.cs

```diff
@@ -354,19 +342,20 @@ void WriteEccLevelAndVersion()
                     default: // M: 00
                         break;
                 }
-                int indexTemp = 2;
-                DecToBin(maskVersion, 3, fStrEcc, ref indexTemp);
+                DecToBin(maskVersion, 3, fStrEcc, 2);
             }
+        }
 
-            void TrimLeadingZeros()
+#if NETCOREAPP
```

Comment:
Why is the #ifdef needed? The attribute should be available on all targets.

Reply:
Well, I should have enabled it on .NET Standard (fixed).

It is not available on .NET Framework prior to 4.5, and QRCoder targets 3.5 and 4.0, not 4.5.  See docs:

https://learn.microsoft.com/en-us/dotnet/api/system.runtime.compilerservices.methodimploptions?view=netframework-4.0

Reply:
Ah, I just looked at the `MethodImpl` itself, not at `AggressiveInlining`. Thanks!